pyacq/devices/webcam_av.py

```diff
@@ -41,7 +41,7 @@ def run(self):
             for frame in packet.decode():
                 arr = frame.to_rgb().to_nd_array()
                 n += 1
-                self.out_stream.send(n, arr)
+                self.out_stream.send(arr, index=n)
```

Comment:
We have an API problem hare because in OutputStream
```
    def send(self, data, index=None, **kargs):
        if index is None:
            index = self.last_index + data.shape[0]
```

This do not work when data.shape[0] is not time.
For webcam data.shape[0] is the width and the index must be obvsiously how many image we get from the beginning.

This is the reason why before this PR index was a mandatory args and not a by default.
This new API is good when shape[0] is time but it is a trap when shape[0] is something else.
So for a Node handling an output stream is better to handle the index directly in the Node and not left the auto counting in OutputStream class that fit very well the must common case shape=nb_sample X nb_channel but so other cases.